src/HearThis/Script/MultiVoiceChapter.cs

```diff
@@ -45,7 +42,18 @@ public MultiVoiceChapter(XElement chapter, MultiVoiceScriptProvider provider)
 
 		private static MultiVoiceBlock[] GetBlocks(MultiVoiceScriptProvider provider, XElement block, ref int index)
 		{
-			var text = block.Element("text")?.Value.Trim() ?? "";
+			string text;
+			switch (provider?.Version?.Major)
```

Comment:
No need to use the Elvis operator on provider. It will crash below if null. (I find that using it when it's not necessary tends to be misleading.)